core/manifest/ConfigStaticManifest.php

```diff
@@ -183,17 +183,15 @@ function getStatics() {
 	}
 
 	/**
-	 * Get the next token to process, incrementing the pointer
+	 * Get the next token to process, incrementing the pointer, skips any whitespace tokens
 	 *
-	 * @param bool $ignoreWhitespace - if true will skip any whitespace tokens & only return non-whitespace ones
 	 * @return null | mixed - Either the next token or null if there isn't one
 	 */
-	protected function next($ignoreWhitespace = true) {
```

Comment:
@tractorcow I was referring to the behavioural change that removing the argument implies rather than the method signature. By removing that argument, we’re now making the function _always_ ignore whitespace with no way to opt out